src/cascadia/TerminalApp/VsDevCmdGenerator.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "VsDevCmdGenerator.h"
+
+std::wstring TerminalApp::VsDevCmdGenerator::GetProfileName(const VsSetupConfiguration::VsSetupInstance instance) const
+{
+    std::wstring name{ L"Developer Command Prompt for VS " };

```

Comment:
Here and with the PowerShell version, you might consider pulling the nickname additional property, which not only depends on `ISetupPropertyStore` being supported but that it's also non-empty. The point of the nickname was to help customers better identify instances when multiple are installed with a moniker they choose. Using that as `(nickname)` instead of `[channel]` if set would be fairly congruent with how shortcuts are displayed (we don't show channels in that case, but I think how you have it here otherwise is nice).